tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -355,7 +395,9 @@ private static void includeCldrJavaScript(HttpServletRequest request, Writer out
 
         out.write(prefix + "jquery.autosize.min.js" + tail); // exceptional
 
-        if (USE_DOJO) {
+        final Boolean doUseDojo = useDojo(request);
+        out.write(String.format("<script>const %s=%s;</script>\n", USE_DOJO_VAR, doUseDojo.toString()));
```

Comment:
Global variable? Maybe OK if very temporary??

Reply:
It's a local, just because it's used twice.

Reply:
Oh, the JavaScript variable, yes, this should be very temporary and removed when DOJO is.